astropy/io/fits/tests/test_connect.py

```diff
@@ -85,11 +85,16 @@ def test_simple_meta_conflicting(self, tmpdir):
         filename = str(tmpdir.join('test_simple.fits'))
         t1 = Table(self.data)
         t1.meta['ttype1'] = 'spam'
-        with catch_warnings() as l:
+        with catch_warnings() as tstwarns:
             t1.write(filename, overwrite=True)
-        assert len(l) == 1
-        assert str(l[0].message).startswith(
-            'Meta-data keyword ttype1 will be ignored since it conflicts with a FITS reserved keyword')
+        if len(tstwarns) != 1:
+            errmsg = os.linesep.join(
+                ['{} from {}:L{}'.format(w.message, w.filename, w.lineno)
```

Comment:
When it is time to clean this up, should we keep this? Or revert all changes to this test? It is nice to keep for future debugging but maybe there won't be necessary future debugging here?

Reply:
I think this should be removed - we have tests to check things work, and need to inspect interactively anyway if they fail.